bst.rb

```diff
@@ -0,0 +1,322 @@
+class String
+  def is_i?
+    /\A[-+]?\d+\z/ === self
+  end
+end
```

Comment:
add comment to tell the functionality of this function , cant understand what it is doing

Reply:
added the discription for the function